src/main/java/io/split/android/client/storage/mysegments/MySegmentsStorage.java

```diff
@@ -1,20 +1,16 @@
 package io.split.android.client.storage.mysegments;
 
-import androidx.annotation.VisibleForTesting;
-
 import java.util.Set;
 
 import io.split.android.client.dtos.SegmentsChange;
+import io.split.android.client.storage.RolloutDefinitionsCache;
 
-public interface MySegmentsStorage {
+public interface MySegmentsStorage extends RolloutDefinitionsCache {
     void loadLocal();
```

Comment:
does it make sense to move `loadLocal` method to the `RolloutDefinitionsCache` interface?